specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json

```diff
@@ -253,6 +253,57 @@
           "description": "Whether the operation can be cancelled.",
           "type": "boolean",
           "readOnly": true
+        },
+        "operationParameters": {
```

Comment:
Are the new properties read-only as the swagger indicates?

Reply:
Yes, they can not be changed.